platform/v1alpha1/platformoperators.crd.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     release.openshift.io/feature-gate: TechPreviewNoUpgrade
+    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
is there a reason not to also include this in single node?  (and eventually i'd expect we have it in hypershift too, but we need to think about that one)

Reply:
Is there any requirement on being "SNO-aware" for using that cluster profile, or are we fine with deploying the HA configuration regardless of cluster topology? I was just keeping the set of changes small, but I don't see any obvious reason we couldn't include the SNO cluster profile annotation too.

Reply:
> Is there any requirement on being "SNO-aware" for using that cluster profile

The expectation for the most part is that you set your replica counts to 1 (not relevant to CRDs obviously).  There may be some components that also tune their resource requests down for the SNO version of their resource yaml, i'm not positive.  Any anti-affinity rules should also be configured to accomodate a SNO install (i.e. ensure that you won't end up with a bunch of unschedulable pods)

My understanding of how profiles work, if you do not add the SNO annotation, the resource will not be deployed on SNO installs at all.

So if you want a user to be able to enable this TP feature on a SNO install, you need to either tag this resource w/ it, or introduce a second yaml that is tagged for SNO.  (Since this is a CRD, i don't see why you'd have a second one.  If it was a deployment that was setting a replica count, i could see having two)

Reply:
Updated in the latest force push and added the SNO cluster profile, and excluded the hypershift (?) cluster profile to match the rest of the platform operator CVO manifests housed in o/platform-operators.